sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py

```diff
@@ -51,6 +51,216 @@ def __init__(self, codes):
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 2), self.getaddr(0x0, 88 + 4 * (2 - 1)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 3), self.getaddr(0x0, 88 + 4 * (3 - 1)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 4), self.getaddr(0x0, 88 + 4 * (4 - 1)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 5), self.getaddr(0x0, 88 + 4 * (5 - 1)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 6), self.getaddr(0x0, 88 + 4 * (6 - 1)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 7), self.getaddr(0x0, 88 + 4 * (7 - 1)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 8), self.getaddr(0x0, 88 + 4 * (8 - 1)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 9), self.getaddr(0x1, 225 + 4 * (9 - 9)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 10), self.getaddr(0x1, 225 + 4 * (10 - 9)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 11), self.getaddr(0x1, 225 + 4 * (11 - 9)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 12), self.getaddr(0x1, 225 + 4 * (12 - 9)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 13), self.getaddr(0x1, 225 + 4 * (13 - 9)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 14), self.getaddr(0x1, 225 + 4 * (14 - 9)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
+                NumberRegField("%s_%d" % (consts.HOST_LANE_COUNT, 15), self.getaddr(0x1, 225 + 4 * (15 - 9)),
+                    *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+                ),
```

Comment:
can we condense HOST_LANE_COUNT similar to ACTIVE_APSEL_CODE  field?

Reply:
No, please see the comments below